source/manual/local-frontend-development.html.md

```diff
@@ -128,6 +128,22 @@ Some components, such as the cookie banner, are pulled in by Static. To test cha
 
 ## Troubleshooting
 
+### JavaScript changes not appearing/JavaScript not working properly
+
+Sometimes the main JavaScript files cannot be served properly when running locally. This happens when Rails automatically inserts fingerprinting into the script URLs when it is not needed e.g. `application-834y2r2hr292y9r.js` is being requested, but only `application.js` exists.
+
+This problem does not currently have a solution, but a workaround is to modify `_layout_for_public.html.erb` in your local `govuk_publishing_components` to hardcode the URLs to the broken JavaScript files (`application.js`, `load-analytics` and any others you need). For example, change:
```

Comment:
Sorry to be nitpicky here, but regarding `This problem does not currently have a solution` -  have we asked the GOVUK devs if they've also experienced this but had a solution for it? Just a bit mindful of documenting  that there's no solution if we haven't had this confirmed.

Reply:
Reasonable point. I've reworded for now.

Reply:
Thanks 👍